packages/odie-client/src/data/use-send-odie-message.ts

```diff
@@ -205,13 +225,30 @@ export const useSendOdieMessage = ( signal: AbortSignal ) => {
 				return;
 			}
 
-			if ( ! odieId ) {
-				addEventToInteraction.mutate( {
-					interactionId: currentSupportInteraction!.uuid,
-					eventData: {
-						event_external_id: returnedChat.chat_id.toString(),
+			const chatId = returnedChat.chat_id;
+			let supportInteraction = currentSupportInteraction;
+
+			try {
+				if ( ! supportInteraction && chatId ) {
+					supportInteraction = await startNewInteraction( {
+						event_external_id: chatId.toString(),
 						event_source: 'odie',
-					},
+					} );
+				} else if ( supportInteraction && ! odieId && chatId ) {
+					supportInteraction = await addEventToInteraction.mutateAsync( {
+						interactionId: supportInteraction.uuid,
+						eventData: {
+							event_external_id: chatId.toString(),
+							event_source: 'odie',
+						},
+					} );
```

Comment:
Sorry for the back and forth, but should we do `else`? Don't we want to create an interaction AND add an event to it?